lib/git_reflow/git_server/pull_request.rb

```diff
@@ -35,13 +53,48 @@ def reviewers_pending_response
         reviewers - approvals
       end
 
+      def check_approvals
+        has_comments_or_approvals = (has_comments? or approvals.any?)
+
+        case self.class.minimum_approvals
+        when "0"
+          true
+        when "", nil
+          # Approvals from every commenter
+          has_comments_or_approvals && reviewers_pending_response.empty?
+        else
+          approvals.size >= self.class.minimum_approvals.to_i and !last_comment.match(self.class.approval_regex).nil?
+        end
+      end
+
+      # Standard Rejection Messages for Error with Reflow Deliver
```

Comment:
@codenamev Just a side note, some of the pre-existing rspec tests are a bit flakey and do not pass 100% of the time. I noticed this recently when I was running all the tests frequently everyday. Not sure if this is much of a concern.